src/quick-lint-js/fe/parse-expression.cpp

```diff
@@ -114,11 +114,17 @@ void parser::visit_expression(expression* ast, parse_visitor_base& v,
     }
     break;
   }
+  case expression_kind::spread: {
+    // expression* child = this->parse_expression(v, precedence{.commas = false});
+    if (ast->child_0()->kind() == expression_kind::_missing) {
+      this->diag_reporter_->report(
+          diag_spread_must_precede_expression{this->peek().span()});
+    }   
+  }
```

Comment:
Please do not implicitly fall through. Either write `[[fallthrough]];` or (better in this case) repeat the code:
```suggestion
    this->visit_expression(ast->child_0(), v, context);
  }
```